mysys/crc32/crc32_arm64.c

```diff
@@ -2,13 +2,13 @@
 #include <string.h>
 #include <stdint.h>
 
+static int pmull_supported;
+
 #if defined(HAVE_ARMV8_CRC)
```

Comment:
Could the definition be moved after the `#ifdef HAVE_ARMV8_CRC`? As far as I remember, we would only use the `pmull` instruction in combination with CRC instructions, not by itself.

Reply:
pmull_supported is referenced in crc32c_aarch64 outside of the HAVE_ARMV8_CRC block, which causes the build error in the first place.